src/.editorconfig

```diff
@@ -166,7 +166,7 @@ dotnet_diagnostic.CA2002.severity = warning
 dotnet_diagnostic.CA2003.severity = none
 dotnet_diagnostic.CA2004.severity = none
 dotnet_diagnostic.CA2006.severity = none
-dotnet_diagnostic.CA2007.severity = error
+dotnet_diagnostic.CA2007.severity = warning
```

Comment:
This should have been a warning from the very beginning, it's less annoying during development. Also during CI we already consider warnings as errors